### ISSUE TITLE
about fix length

### ISSUE THREAD

**Issue by pariswu1988** [omarwagih/rmotifx#6]
hi, could please help how to make fix length sequence?

**Comment by cjjong23** [comment]
Yes, does anyone have any suggestions on making the same width sequences? And, where do you guys get the human reference database?

**Comment by arunhpatil** [comment]
For proteomics data, you could refer to [PTM-Pro](http://ptm-pro.inhouseprotocols.com/) . In FAQs you can download protein databases of few model organsims and can also get fix length sequence upon using the tool. let me know if you need any further information.

**Comment by pariswu1988** [comment]
> For proteomics data, you could refer to [PTM-Pro](http://ptm-pro.inhouseprotocols.com/) . In FAQs you can download protein databases of few model organsims and can also get fix length sequence upon using the tool. let me know if you need any further information.

hi, yes, I can find the databases of arabidopsis but ca not find the tool to get fix length. That will be very kind of you if give this info. Thanks a lot!

**Comment by arunhpatil** [comment]
Sure. Can you send me a sample file with protin accession and the site of modification. I will setup a module to fetch the fixed lenth window of 15 aa.

**Comment by cjjong23** [comment]
Hi Arun,

    Thanks for the info. I have a question though, I am searching for human
protein databases. Which one in the databases would you recommend? Thank
you so much.

**Comment by arunhpatil** [comment]
Hi. According to literature, neXtProt <https://www.nextprot.org/>and  RefSeq
<https://www.ncbi.nlm.nih.gov/refseq/> are widely used databases in the
community. Both are updated frequently. At PTM-Pro, we have the recent
version RefSeq 92, you may use that. You can download databases currently
indexed on the server from here.
<http://ptm-pro.inhouseprotocols.com/Databases/>

Arun.

**Comment by cjjong23** [comment]
Hi Arun,

    I am sorry to keep bothering you. Since I am interested in using the
rmotifx, is there a way I can pre-align all the background sequences? Thank
you so much again for your advice.

**Comment by arunhpatil** [comment]
Hi.

You could download the pre-aligned data from phosphosite plus
<https://www.phosphosite.org/homeAction.action> for your organism of
interest and extract the PTM-window.

I hope this helps.

Arun.

**Comment by arunhpatil** [comment]
Hi,

We have updated the version of PTM-Pro
<http://ptm-pro.inhouseprotocols.com/>, a software tool to analyze multiple
post-translational modifications. The second version (PTM-Pro2.0) includes
lot of additional features. Check out and share accross with the members of
your Institute, group, and friends for an extended reach for the use of
PTM-Pro and to further improve this user-friendly open-source tool.

URL: http://ptm-pro.inhouseprotocols.com/

If you have any queries or suggestions please write to
support@inhouseprotocols.com.

In case, If you use the PTM-Pro tool, plesae cite the following:
Patil, A. H., Datta, K. K., * et al*., Dissecting Candida pathobiology:
Post-translational modifications on the *Candida tropicalis* proteome.
2018. * OMICS: A Journal of Integrative Biology*. [PubMed].
<https://www.ncbi.nlm.nih.gov/pubmed/30106353>

Thank you,
Arun.

**Comment by omarwagih** [comment]
@pariswu1988 you can use the `msa` package: https://bioconductor.org/packages/release/bioc/html/msa.html to pre-align your sequences.
